ydb/core/blobstorage/pdisk/blobstorage_pdisk_config.h

```diff
@@ -127,6 +127,9 @@ struct TPDiskConfig : public TThrRefBase {
 
     ui64 ExpectedSlotCount = 0;
 
+    // Free chunk percent that triggers Cyan color (e.g. 100 is 10%). Between 130 (default) and 13.
```

Comment:
```suggestion
    // Free chunk permille that triggers Cyan color (e.g. 100 is 10%). Between 130 (default) and 13.
```


https://en.wikipedia.org/wiki/Per_mille